contracts/DAORegistry.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Would be great to have some tests for this contract as well, also for understanding the contract.